plugin/nvim-workbench.vim

```diff
@@ -8,13 +8,13 @@ endfunction
 " Toggle checkbox
 " - [ ] testing -> - [x] testing
 " - [x] testing -> - [ ] testing
-nnoremap <expr> <silent> <Plug>WorkbenchToggleCheckbox <SID>SearchCheck() ? ':s/\[.*\]/\[x\]<CR>': ':s/\[x\]/\[ \]<Enter>' 
+nnoremap <expr> <silent> <Plug>WorkbenchToggleCheckbox <SID>SearchCheck() ? ':s/\[.*\]/\[x\]<CR>': ':s/\[x\]/\[ \]<Enter>/<bar>noh<CR>' 
```

Comment:
Does this only works if `SearchCheck()` returns false? We might want to add to the first part? When `SearchCheck()` returns true.

Reply:
No, it doesn't work actually.